cristin-import/src/test/java/cucumber/ArtisticFeatures.java

```diff
@@ -110,4 +104,221 @@ public void theCristinResultLackTheDurationInBothMetadataFields() {
+        var audioVisualManifestation = (AudioVisualPublication) audioVisualManifestationOptional.get();
+        assertThat(audioVisualManifestation.getMediaType().getType().getValue(), is(equalTo(expectedMediaSubType)));
+        assertThat(audioVisualManifestation.getIsrc(), is(equalTo(new Isrc(expectedIsrc))));
+        var actualPublisher = (UnconfirmedPublisher) audioVisualManifestation.getPublisher();
+        assertThat(actualPublisher.getName(), is(equalTo(expectedUnconfirmedPublisher)));
+    }
+
+    @And("the performance has a ISMN equal to {string}")
+    public void thePerformanceHasAIsmnEqualTo(String ismn) {
+        scenarioContext.getCristinEntry().getCristinArtisticProduction().setIsmn(ismn);
+    }
+
+    @Then("the NVA resource has a MusicScore")
+    public void theNvaResourceHasAMusicScore() {
+        var musicalWorkPerformance = extractMusicPerformance();
+        assertThat(musicalWorkPerformance.getManifestations(), hasItem(instanceOf(MusicScore.class)));
+    }
+
+    @And("the MusicScore has a ISMN equal to {string}")
+    public void theMusicScoreHasAnIsmnNEqualTo(String ismn) throws InvalidIsmnException {
+        var musicalWorkPerformance = extractMusicPerformance();
+        var musicScoreOptional = musicalWorkPerformance
+            .getManifestations()
+            .stream()
+            .filter(manifestation -> manifestation instanceof MusicScore)
+            .findFirst();
+        assertThat(musicScoreOptional.isPresent(), is(equalTo(true)));
+        var musicScore = (MusicScore) musicScoreOptional.get();
+        assertThat(musicScore.getIsmn(), is(equalTo(new Ismn(ismn))));
+    }
+
+    @And("the performance has a ensemble name equal to {string}")
+    public void thePerformanceHasAEnsembleNameEqualTo(String ensembleName) {
+        scenarioContext.getCristinEntry().getCristinArtisticProduction().setEnsembleName(ensembleName);
+    }
+
+    @And("the MusicScore has ensemble equal to {string}")
+    public void theMusicScoreHasEnsembleEqualTo(String ensemble) {
+        var musicalWorkPerformance = extractMusicPerformance();
+        var musicScoreOptional = musicalWorkPerformance.getManifestations()
+            .stream()
+            .filter(manifestation -> manifestation instanceof MusicScore)
+            .findFirst();
+        assertThat(musicScoreOptional.isPresent(), is(equalTo(true)));
+        var musicScore = (MusicScore) musicScoreOptional.get();
+        assertThat(musicScore.getEnsemble(), is(equalTo(ensemble)));
+    }
+
+    private String readResourceFile(Path path) {
+        return IoUtils.stringFromResources(path);
+    }
+
+    private CristinProduct readProductFromResource() {
+        var cristinProductString = readResourceFile(Path.of("type_produkt.json"));
+        return attempt(
+            () -> JsonUtils
+                .dtoObjectMapper
+                .readValue(cristinProductString, CristinProduct.class))
+            .orElseThrow();
+    }
```

Comment:
Why does code style wrap it like this

Reply:
Looking better now?

To answer you question:
Because code-style is broken on my computer. Reloading the UnitStyle does not work.